Python/Product/Analysis/Interpreter/Ast/AstScrapedPythonModule.cs

```diff
@@ -25,19 +27,27 @@
 
 namespace Microsoft.PythonTools.Interpreter.Ast {
     class AstScrapedPythonModule : IPythonModule {
-        private readonly string _filePath;
+        private readonly string _name, _filePath;
+        private readonly bool _isBuiltins;
```

Comment:
Would this make more sense as a derived class and a bunch of virtual/override? It seems like all the `_isBuiltins` checks are pre/post method body, except for building the scraper command line, which can be factored out into its own method.